setup.cfg

```diff
@@ -35,6 +35,27 @@ project_urls =
 [options]
 python_requires = >=3.7
 
+[options.extras_require]
+docs =
+    olefile
```

Comment:
I feel like I should be able to work this out, but I can't - why is olefile needed for docs?

Reply:
Without it the autodoc fails for `FpxImagePlugin` and `MicImagePlugin`:

```
WARNING: autodoc: failed to import module 'FpxImagePlugin' from module 'PIL'; the following exception was raised:
No module named 'olefile'
looking for now-outdated files... WARNING: autodoc: failed to import module 'MicImagePlugin' from module 'PIL'; the following exception was raised:
No module named 'olefile'
...
/home/docs/checkouts/readthedocs.org/user_builds/pillow-hugovk/checkouts/extras_require/docs/reference/plugins.rst:60: WARNING: py:mod reference target not found: PIL.FpxImagePlugin
/home/docs/checkouts/readthedocs.org/user_builds/pillow-hugovk/checkouts/extras_require/docs/reference/plugins.rst:164: WARNING: py:mod reference target not found: PIL.MicImagePlugin
```

https://readthedocs.org/api/v2/build/16137387.txt
https://readthedocs.org/projects/pillow-hugovk/builds/16137387/

And iirc we just get the header and not this:

![image](https://user-images.githubusercontent.com/1324225/155102651-98d43386-5f15-42c5-b69f-f91725846216.png)

Reply:
Thanks